src/qrl/core/messagereceipt.py

```diff
@@ -17,6 +17,20 @@
 # FIXME: Refactor / improve
 class MessageReceipt(object):
     """
+    MessageReceipt is a hash map that looks like this:

```

Comment:
Better to talk about the format at the end of the description. Usually documentation of a class should start with an introduction, before you directly jump into the format.